tests/scripts/collect-logs.sh

```diff
@@ -27,32 +27,36 @@ for NAMESPACE in "${NAMESPACES[@]}"; do
   mkdir "${NS_DIR}"
 
   # describe every one of the k8s resources in the namespace which rook commonly uses
-  for KIND in 'pod' 'deployment' 'job' 'daemonset' 'secret' 'cm'; do
-    kubectl -n "$NAMESPACE" get "$KIND" -o wide > "${NS_DIR}"/"$KIND"-list.txt
+  for KIND in 'pod' 'deployment' 'job' 'daemonset' 'cm'; do
+    kubectl -n "$NAMESPACE" get "$KIND" -o wide >"${NS_DIR}"/"$KIND"-list.txt
     for resource in $(kubectl -n "$NAMESPACE" get "$KIND" -o jsonpath='{.items[*].metadata.name}'); do
-      kubectl -n "$NAMESPACE" describe "$KIND" "$resource" > "${NS_DIR}"/"$KIND"-describe--"$resource".txt
+      kubectl -n "$NAMESPACE" describe "$KIND" "$resource" >"${NS_DIR}"/"$KIND"-describe--"$resource".txt
 
       # collect logs for pods along the way
       if [[ "$KIND" == 'pod' ]]; then
-        kubectl -n "$NAMESPACE" logs --all-containers "$resource" > "${NS_DIR}"/logs--"$resource".txt
+        kubectl -n "$NAMESPACE" logs --all-containers "$resource" >"${NS_DIR}"/logs--"$resource".txt
       fi
     done
   done
 
+  for secret in $(kubectl -n "$NAMESPACE" get secrets -o jsonpath='{.items[*].metadata.name}'); do
+    kubectl -n "$NAMESPACE" get -o yaml secret "$secret" >"${NS_DIR}"/secret-describe--"$secret".txt
```

Comment:
```suggestion
    kubectl -n "$NAMESPACE" get -o yaml secret "$secret" > "${NS_DIR}"/secret-describe--"$secret".txt
```

Reply:
my vscode auto-save removing the space so keeping it, let me know if we need a space, I see vscode adding space before `>` but not after `>`